### ISSUE TITLE
"Internal server error: Unknown" While restoring back up file from beta version to latest version.

### ISSUE THREAD

**Issue by naymintun800** [hiddify/Hiddify-Manager#4605]
# Internal Error Stacktrace:
**Error Message**:   Value ssh_host_rsa_pk is not found in this enum type declaration
```
Traceback (most recent call last):
  File "/opt/hiddify-manager/.venv/lib/python3.10/site-packages/flask/app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
  File "/opt/hiddify-manager/.venv/lib/python3.10/site-packages/flask/app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "/opt/hiddify-manager/.venv/lib/python3.10/site-packages/flask_classful.py", line 303, in proxy
    response = view(**request.view_args)
  File "/opt/hiddify-manager/.venv/lib/python3.10/site-packages/hiddifypanel/auth.py", line 130, in decorated_view
    return fn(*args, **kwargs)
  File "/opt/hiddify-manager/.venv/lib/python3.10/site-packages/flask_classful.py", line 271, in inner
    return fn(*args, **kwargs)
  File "/opt/hiddify-manager/.venv/lib/python3.10/site-packages/hiddifypanel/panel/admin/Backup.py", line 44, in post
    hiddify.set_db_from_json(json_data,
  File "/opt/hiddify-manager/.venv/lib/python3.10/site-packages/hiddifypanel/panel/hiddify.py", line 235, in set_db_from_json
    bulk_register_configs(json_data["hconfigs"], commit=True, override_unique_id=override_unique_id)
  File "/opt/hiddify-manager/.venv/lib/python3.10/site-packages/hiddifypanel/models/config.py", line 172, in bulk_register_configs
    add_or_update_config(commit=False, child_id=child_id, **conf)
  File "/opt/hiddify-manager/.venv/lib/python3.10/site-packages/hiddifypanel/models/config.py", line 156, in add_or_update_config
    ckey = ConfigEnum(c)
  File "/opt/hiddify-manager/.venv/lib/python3.10/site-packages/fast_enum/fastenum.py", line 158, in get
    raise ValueError(f'Value {val} is not found in this enum type declaration')

```

## Details:
**Hiddify Version**: 10.70.9
**Python Version**:  3.10.12 (main, Sep 11 2024, 15:47:36) [GCC 11.4.0]
**OS**:              Linux-5.15.0-125-generic-x86_64-with-glibc2.35
**User Agent**:      Unknown

**Comment by litoosh13** [comment]
check out here :
https://t.me/hiddify_board/482495/518026

**Comment by naymintun800** [comment]
> check out here : https://t.me/hiddify_board/482495/518026

Help, Your bash script doesn't fix the problem "Internal server error: Unknown". I've tried rebulding and reinstalling hiddify manager.

**Comment by litoosh13** [comment]
you should just bring back your users and no need to bring your settings

**Comment by naymintun800** [comment]
> you should just bring back your users and no need to bring your settings

What about admin paths and user paths?

**Comment by litoosh13** [comment]
Yes, after importing backup, you can manually set your old settings.

**Comment by naymintun800** [comment]
Looks like something wrong with subscription link or user path. In the "Very Advanced Settings" I only changed "Proxy Path for Admins" and "Proxy Path for Clients" to old settings. Do I need to change anything else in the settings?

**Comment by naymintun800** [comment]
> no but do this in order to be sure:
> 
> 1. use hiddify next for client
> 2. set core in advance setting on singbox
>    if with this two step still you are facing problem
>    rebuild the server and install again
>    make sure you have backup

I've tried everything but still doesn't work on Iphones with "Streisand" since Hiddify client app is not on appstore, old hiddify manager version work on streisand fine tho. For the record I've tried rebuilding server and installing hiddify manager 3 times and I get the same result. I'm really tired I've been doing this since morning none stop. Is there gonna be any release on version update for hiddify manager panel soon that'll fix all of these problems ?
